docs/pages/access-controls/reference.mdx

```diff
@@ -435,3 +435,17 @@ spec:
 
 Refer to the [Second Factor - U2F guide](./guides/u2f.mdx) if you have a cluster
 using the legacy U2F support.
+
+## Filter fields explained
+
+Explanation of the fields used in this guide for the `where` and `filter` conditions.
+
+| Field                      | Description                                         |
+|----------------------------|-----------------------------------------------------|
+| `user.roles`               | the `list` of roles assigned to a user              |
```

Comment:
```suggestion
| `user.roles`               | The list of roles assigned to a user              |
```
I don't think `list` needs to be in code style here since it refers to a type rather than a specific keyword used in a config resource.